src/main/java/seedu/address/ui/CompletableDeadlineCard.java

```diff
@@ -34,6 +36,7 @@ public CompletableDeadlineCard(CompletableDeadline deadline, int displayedIndex)
         id.setText(displayedIndex + ". ");
         description.setText(deadline.getDescription());
         date.setText(DateUtil.decodeDate(deadline.getBy()));
+        completedLabel.setText(deadline.getIsDone() ? "✔" : "");
```

Comment:
Do you wanna simplify "deadline.getIsDone() ? "✔" : """ with a method in Deadline? Or even from toString?

Reply:
yeap good catch, this should be changed

Reply:
Actually, now that i think it i think it's ok to leave it like this. I feel CompletableDeadlineCard should deal with UI elements (what to display when the item is completed), and Todo class should stick to containing fields related to a todo object (boolean value of whether it's done or not).

Reply:
This so in the future if we want to change how a completed todo is supposed to look we just need to change the UI

Reply:
Sounds good to me!